### PR TITLE
adding redirect rules to prevent google links from reaching page not …

### PR DESCRIPTION
**Pull request by chance19**
…found

Signed-off-by: Chance Whittaker <chance19@gmail.com>

Currently some links from Google, such as https://docs.cert-manager.io/en/release-0.11/ would route to https://cert-manager.io/docs/en/release-0.11/ which would take the browser to a 404 page not found (see screenshot)
image

This change should ensure that these links are at least routed back to the specific release-note page that was referenced in the link. This is not a full solution to the problem just an improvement on the current situation.

### CODE REVIEW

**Comment by chance19** [comment]
CC @mattbates for visibility

**Comment by chance19** [comment]
Three commits:

1. Adding more specific redirect rules to mitigate 404 errors
2. Updating outdated links in content
3. Adding specific links from outside sources (https://www.fairwinds.com/blog/upgrade-cert-manager-its-worth-it) that needed to be redirected and applying contextual comments

**Comment by SgtCoDFish** [comment]
Happy to review this once the conflicts have been resolved :+1:

File: _redirects [hunk 1]
```diff
@@ -36,7 +36,30 @@ https://cert-manager.io/docs/usage/kubectl-plugin/#completion https://cert-manag
...
+https://docs.cert-manager.io/en/release-0.3/* https://cert-manager.io/docs/release-notes/release-notes-0.3/ 301!
+https://docs.cert-manager.io/en/release-0.4/* https://cert-manager.io/docs/release-notes/release-notes-0.4/ 301!
+https://docs.cert-manager.io/en/release-0.5/* https://cert-manager.io/docs/release-notes/release-notes-0.5/ 301!
+https://docs.cert-manager.io/en/release-0.6/* https://cert-manager.io/docs/release-notes/release-notes-0.6/ 301!
+https://docs.cert-manager.io/en/release-0.7/* https://cert-manager.io/docs/release-notes/release-notes-0.7/ 301!
+https://docs.cert-manager.io/en/release-0.8/* https://cert-manager.io/docs/release-notes/release-notes-0.8/ 301!
+https://docs.cert-manager.io/en/release-0.9/* https://cert-manager.io/docs/release-notes/release-notes-0.9/ 301!
+https://docs.cert-manager.io/en/release-0.10/* https://cert-manager.io/docs/release-notes/release-notes-0.10/ 301!
+https://docs.cert-manager.io/en/release-0.11/* https://cert-manager.io/docs/release-notes/release-notes-0.11/ 301!
+https://docs.cert-manager.io/en/release-0.13/* https://cert-manager.io/docs/release-notes/release-notes-0.12/ 301!
```
Comment (SgtCoDFish) [inline _redirects:57]: suggestion: should this be the following:

```suggestion
https://docs.cert-manager.io/en/release-0.12/* https://cert-manager.io/docs/release-notes/release-notes-0.12/ 301!
```

?

**Review by SgtCoDFish — CHANGES_REQUESTED** [review]
This looks good, there's just one little thing I've raised.

There are a _lot_ of references to `docs.cert-manager.io` if I do ` grep -R "docs\.cert-manager\.io" .` in the website repo, but most are in old docs and they should be caught by the redirect rules we're adding here :+1:

File: _redirects (hunk 1 above)
Comment (chance19) [reply _redirects]: Thanks for catching this. You are absolutely right. I have made the change in the most recent commit.

**Review by SgtCoDFish — APPROVED** [review]
/lgtm

Good stuff :grin:

**Pull request merged**